docs/tutorial.rst

```diff
@@ -197,6 +199,7 @@ Now let's add the new test to test_search.py:
 
 .. literalinclude:: code/round_2/test_search.py
     :language: python
+    :emphasize-lines: 3, 10-15, 23-27
```

Comment:
No code shows up https://github.com/billagee/bok-choy/blob/doc-edits/docs/tutorial.rst#add-the-new-test